example/performance/README.md

```diff
@@ -0,0 +1,106 @@
+# Performance
+
+Palisade includes a performance tool for testing some simple scenarios. It uses the example data and generates some fake HR data using
+the [HR data generator](../hr-data-generator/README.md).
+
+## Usage
+
+### Building
+To use the performance tool, you must first build Palisade (from the Palisade root directory):
+```bash
+mvn clean install -P example
+./example/deployment/local-jvm/bash-scripts/buildServices.sh
+```
+
+Open a separate terminal window and start the Palisade services running on your local machine:
```

Comment:
Can we reduce the changing of terminal windows, e.g. do as much as possible in the first before swapping to a second.

Reply:
Done